src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const spans = document.querySelectorAll('.population');
+
+let sum = 0;
+
+spans.forEach((span) => {
+  const number = parseFloat(span.innerText.replace(/,/g, ''));
+
+  if (!isNaN(number)) {
+    sum += number;
+  }
+});
+
+const average = sum / spans.length;
```

Comment:
The calculation of the average might be incorrect if there are elements with non-numeric values. Consider checking if `spans.length` is zero or using a counter for valid numbers to avoid division by zero or incorrect averages.